hadoop-ozone/dist/src/main/compose/upgrade/compose/ha/docker-compose.yaml

```diff
@@ -82,7 +82,7 @@ services:
     env_file:
       - docker-config
     environment:
-      HADOOP_CONF_DIR: /opt/hadoop/etc/hadoop
+      OZONE_CONF_DIR: /opt/hadoop/etc/hadoop
```

Comment:
This service runs `hadoop kms`, which is Hadoop, not Ozone.  This occurrence should not be changed.